### PR TITLE
Update hwameistor/resources/expand

### PR DESCRIPTION
**Pull request by WANG0608GitHub**
- Update Chinese page to keep in line with UI
- Update English to keep in line with Chinese and UI

### CODE REVIEW

**Comment by WANG0608GitHub** [comment]
en preview: https://deploy-preview-5591--daocloud-docs.netlify.app/en/storage/hwameistor/resources/expand

File: docs/zh/docs/storage/hwameistor/resources/expand.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ HwameiStor 支持 `CSI 卷扩容` 。这个功能实现了通过修改 `PVC` 的
 
 ## 手动扩容数据卷
 
-1. 进入对应集群，选择 **存储** -> **Hwameistor**
+1. 进入对应集群，选择 **容器存储** -> **Hwameistor**
```
Comment (windsonsea) [inline docs/zh/docs/storage/hwameistor/resources/expand.md:7]:
```suggestion
1. 进入对应集群，选择 **容器存储** -> **HwameiStor**
```

File: docs/zh/docs/storage/hwameistor/resources/expand.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ HwameiStor 支持 `CSI 卷扩容` 。这个功能实现了通过修改 `PVC` 的
...
 
-2. 点击 **本地卷** ，在本地卷列表界面，选择一条本地卷进行`扩容`操作
+2. 点击 **本地卷** ，在本地卷列表界面，选择一条本地卷，点击 **⋮**， 进行 **扩容** 操作
```
Comment (windsonsea) [inline docs/zh/docs/storage/hwameistor/resources/expand.md:9]:
```suggestion
2. 点击 **本地卷** ，在本地卷列表中，点击一个本地卷右侧的 **┇**， 选择 **扩容**
```

File: docs/en/docs/storage/hwameistor/resources/expand.md [hunk 3]
```diff
@@ -5,19 +5,19 @@ modifying the size of `PVC`. Currently, it supports both manual and automatic ex
 
 ## Manual Expansion
 
-1. Access the proper cluster and navigate to **Storage** -> **Hwameistor**
+1. Access the proper cluster and navigate to **Container Storage** -> **Hwameistor**
 
-2. Click **Local Volume** . In the local volume list interface, select a local volume to perform the `expand` operation.
+2. Click **Local Volumes** . In the local volume list interface, click **⋮** to perform the **Expand** operation.
```
Comment (windsonsea) [inline docs/en/docs/storage/hwameistor/resources/expand.md:10]:
```suggestion
2. Click **Local Volumes** . On the right side of the local volume list, click **┇** and select **Expand** .
```

File: docs/en/docs/storage/hwameistor/resources/expand.md [hunk 4]
```diff
@@ -5,19 +5,19 @@ modifying the size of `PVC`. Currently, it supports both manual and automatic ex
 
 ## Manual Expansion
 
-1. Access the proper cluster and navigate to **Storage** -> **Hwameistor**
+1. Access the proper cluster and navigate to **Container Storage** -> **Hwameistor**
```
Comment (windsonsea) [inline docs/en/docs/storage/hwameistor/resources/expand.md:8]:
```suggestion
1. Access the proper cluster and navigate to **Container Storage** -> **HwameiStor**
```

File: docs/en/docs/storage/hwameistor/resources/expand.md [hunk 5]
```diff
@@ -41,12 +41,12 @@ spec:
...
-  meaning that after expansion, the volume usage will be below this percentage.
-- `resizeThreshold`: Indicates a usage ratio. When the volume usage reaches this percentage,
-  the expansion action will be triggered.
-- `nodePoolUsageLimit`: Represents the upper limit of node storage pool usage.
-  If the usage of a pool reaches this percentage, the volumes in this pool will not be automatically expanded.
+- `warningThreshold` is currently not associated with any alert action. It serves as a target ratio, meaning that
+  after expansion, the volume usage will be below this percentage.
+- `resizeThreshold` indicates a usage ratio. When the volume usage reaches this percentage, the expansion action
+  will be triggered.
+- `nodePoolUsageLimit` represents the upper limit of node storageclass usage. If the usage of a storageclass reaches
```
Comment (windsonsea) [inline docs/en/docs/storage/hwameistor/resources/expand.md:48]:
```suggestion
- `nodePoolUsageLimit` represents the upper limit of node StorageClass usage. If the usage of a StorageClass reaches
```
连起来需要用驼峰写法，参见 https://kubernetes.io/docs/concepts/storage/storage-classes/

**Review by windsonsea — APPROVED** [review]

**Pull request merged**